agent/tool-scripts/pprof

```diff
@@ -146,7 +160,12 @@ ose_pprof() {
 
 collect_data() {
     while true; do
-            $tool_bin tool $tool -text -seconds=$interval -output $PPROF_TMPDIR/$(date +%Y-%m-%d:%H:%M:%S)_.txt http://localhost:6060/debug/pprof/profile
+        if [ $(systemctl list-unit-files | grep atomic-openshift-master.service | awk {'print $2'}) == "masked" ]; then
+            $tool_bin tool $tool -text -seconds=$interval -output $PPROF_TMPDIR/$(date +%Y-%m-%d:%H:%M:%S)_controller.txt http://localhost:6060/debug/pprof/profile &
```

Comment:
@ekuric, @vikaslaad, so first, with the ampersands present (causing each invocation of the pprof tool to run in the background, won't that spin in a loop creating multiple background processes?

Have you considered what other tools do, like sar-tools, which create a background process, save the pid in a file so that the stop tools operations can use that pid to kill the process?

It would seem that the `collect_data()` function is what you want to run in the background, where it does not background the pprof tools.  So then we'd have a `wait` called after `collect_data()` on line 178 which is put in the background.